policy/flask/flask_documentation.md

```diff
@@ -0,0 +1,1664 @@
+
+**execstack** - Make the main process stack executable.
+
+**fork** - Fork into two processes.
+
+**getattr** - Get attributes of a file.
+
+**getcap** - Get Linux capabilities.
+
+**getpgid** - Get group Process ID of a process.
+
+**getrlimit** - Get process rlimit information.
+
+**getsched** - Get priority of a process.
+
+**getsession** - Get session ID of another process.
+
+**noatsecure** - Disable secure mode environment cleansing (AT\_SECURE).
+
+**ptrace** - Attach to another process for tracing.
+
+**rlimitinh** - Inherit resource limits from the caller.
+
+**setcap** - Set Linux capabilities.
+
+**setcurrent** - Set the current process context.
+
+**setexec** - Override the default context for the next exec().
+
+**setfscreate** - Override the default context for file creation.
+
+**setkeycreate** - Override the default context for key creation.
+
+**setpgid** - Set group Process ID of a process.
+
+**setrlimit** - Change process hard limits.
+
+**setsched** - Set priority of a process.
+
+**setsockcreate** - Override the default context for socket creation.
+
+**share** - Allow state sharing with cloned or forked processes.
+
+**sigchld** - Send SIGCHLD signal.
+
+**siginh** - Inherit signal state from caller.
+
+**sigkill** - Send SIGKILL signal.
+
+**signal** - Send a signal other than SIGKILL, SIGSTOP, or SIGCHLD.
+
+**signull** - Test for the existence of another process without sending a signal.
+
+**sigstop** - Send SIGSTOP signal.
+
+**transition** - Transition to a new context on exec().
+
+## class process2
+
+**nnp\_transition** - Transition to a more privileged domain even if no\_new\_privs kernel flag is set.
+
+**nosuid\_transition** - Transition to a new domain based on setuid flag even if the file system is marked nosuid.
+
+## class security
+
+This is the security server object and there is only one instance of this object (for the SELinux security server).
+
+**check\_context** - Write context in selinuxfs.
+
+**compute\_av** - Compute an access vector given a source/target/class.
+
+**compute\_create** - Get create info in selinuxfs.
+
+**compute\_member** - Determines the context to use when selecting a member of a polyinstantiated object.
+
+**compute\_relabel** - Get relabel info in selinuxfs.
+
+**compute\_user** - Get user info in selinuxfs.
+
+**load\_policy** - Load the security policy.
+
+**read\_policy** - Read the kernel policy to userspace.
+
+**setbool** - Set a boolean value.
+
+**setcheckreqprot** - Set if SELinux will check original protection mode or modified protection mode (read-implies-exec) for mmap/mprotect.
+
+**setenforce # was avc\_toggle in system class** - Change the enforcement state of SELinux.
+
+**setsecparam** - Set kernel access vector cache tuning parameters.
+
+**validate\_trans** - Validate a transition. (This determines whether a transition from scone to newcon using tcon as the target for object class tclass is valid in the loaded policy.)
```

Comment:
scone -> scon
Also, we might want to highlight (e.g. use italics) "scon", "tcon" and "tclass" to make this more readable.